centrapay/centrapay-docs
# Upgrade Astro to v3.0

Upgrade Astro to v3.0 using upgrade guides. Note that all astro integrations have also been upgraded as per the upgrade guide. Doing these separately would have resulted in unmet peer dependencies.
- [Astro v3.0 upgrade guide](https://docs.astro.build/en/guides/upgrade-to/v3/#overview)
- [@astrojs/mdx changelog](https://github.com/withastro/astro/blob/main/packages/integrations/mdx/CHANGELOG.md)
- [@astrojs/tailwind changelog](https://github.com/withastro/astro/blob/main/packages/integrations/tailwind/CHANGELOG.md)
- [@astrojs/vue changelog](https://github.com/withastro/astro/blob/main/packages/integrations/vue/CHANGELOG.md)

See also:
- [Astro v3 annoucement](https://astro.build/blog/astro-3/)
- [Astro v2.9 introduced redirect support](https://docs.astro.build/en/core-concepts/routing/#redirects)

[See our dev environment for a live preview here](http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com)

Test plan:
- Confirm content is being rendered correctly from our MDX files
- Confirm custom components are being rendered as expected from our MDX files
- Confirm correct styles are being applied
- Confirm sidebar navigation working as expected
- Confirm local development server and static preview loads site at Astro's new port (4321)
